kimchi/src/precomputed_srs.rs

```diff
@@ -62,17 +64,15 @@ mod tests {
 
     #[test]
     fn test_regression_serialization_srs_with_generators_vesta() {
-        // This is the same as Pallas as we encode the coordinate x only.
-        // Generated with commit 4c69a4defdb109b94f1124fe93283e728f1d8758
-        let exp_output = "9291c421010000000000000000000000000000000000000000000000000000000000000000c421010000000000000000000000000000000000000000000000000000000000000000";
+        // Generated with commit 309e1f1d4bc76fd37bc5403f7c728238f6a39aee
+        let exp_output = "9391c4410100000000000000000000000000000000000000000000000000000000000000626a27b0b989434e0ca28b297f3aceac4c7589ae3a4eb6136bae22a96e66431900c4410100000000000000000000000000000000000000000000000000000000000000626a27b0b989434e0ca28b297f3aceac4c7589ae3a4eb6136bae22a96e6643190080";
         test_regression_serialization_srs_with_generators::<Vesta>(exp_output.to_string())
     }
 
     #[test]
     fn test_regression_serialization_srs_with_generators_pallas() {
-        // This is the same as Vesta as we encode the coordinate x only.
-        // Generated with commit 4c69a4defdb109b94f1124fe93283e728f1d8758
-        let exp_output = "9291c421010000000000000000000000000000000000000000000000000000000000000000c421010000000000000000000000000000000000000000000000000000000000000000";
+        // Generated with commit 309e1f1d4bc76fd37bc5403f7c728238f6a39aee
+        let exp_output = "9391c4410100000000000000000000000000000000000000000000000000000000000000bb2aedca237acf1971473d33d45b658f54ee7863f0a9df537c93120aa3b5741b00c4410100000000000000000000000000000000000000000000000000000000000000bb2aedca237acf1971473d33d45b658f54ee7863f0a9df537c93120aa3b5741b0080";
```

Comment:
I don't really like we change the encoding. Can we not keep the same?

Reply:
The difference is that now we serialize the Lagrange bases as well, so this extra tail is that.